code.py

```diff
@@ -5,24 +5,31 @@
 from adafruit_ble.services.nordic import UARTService
 
 ble = BLERadio()
+ble.name = "Raptor 3"
 
-uart_connection = None
+uart_connections = []
+MAX_CONNECTIONS = 1
 
 while True:
-    ble.name = "Raptor 3"
-    if not uart_connection:
-        print("Trying to connect...")
+    if len(uart_connections) < MAX_CONNECTIONS:
```

Comment:
If you don't want any max number of devices then simply remove the if statements here.